automated-comments/go/hamming/mixture_of_runes_and_bytes.md

```diff
@@ -0,0 +1,3 @@
+Beware that you are using a mixture of `runes` and `bytes` here. This is working fine as the exercise only
+requires you to handle `ascii` characters. You can go ahead and add a few more test cases containing non-ascii characters.
+See what happens!
```

Comment:
We should be consistent about how we refer to ASCII. We've used:

- ascii
- ASCII
- `ascii`

To tighten this up, maybe:

> Using a mixture of `runes` and `bytes` is safe here, because the input only uses ASCII characters. To see what happens when it's not safe, try adding a test that uses non-ASCII characters.